pyproject.toml

```diff
@@ -0,0 +1,199 @@
+# https://packaging.python.org/en/latest/specifications/pyproject-toml/
+[build-system]
+requires = ["pdm-backend", "wheel>=0.42"]
```

Comment:
It's needed to build the wheels but not the sdist. Why do you think it will be an issue to package it with Linux distribution? `python-wheel` is part of the new build workflow, cut off from `setuptools`, I am sure it's packaged for all the distros.

Reply:
We at Gentoo don't like declaring dependencies when they're not actually needed. Isn't there a better way to declare it as a dependency only for the wheels? Ping @mgorny.

Reply:
Ok, fair enough.
The dependencies defined in `build-system` are used when you do `pip install .`
In this case you don't need to build a wheel with the libs bundled. So you can remove `wheel` from these deps.
What can be done then, is to put the `wheel` dep in a pdm script that builds the bundled wheels (I'm preparing this PR).

Reply:
#141